src/LoginComponent.js

```diff
@@ -36,7 +40,7 @@ class LoginComponent extends React.Component {
 			measurementId: "G-MDTYYPRVDF"
 		};
 		// Initialize Firebase
-		firebase.initializeApp(firebaseConfig);
+		!firebase.apps.length ? firebase.initializeApp(firebaseConfig) : firebase.app();
```

Comment:
what is this doing? like if the length of the db is not 0 then do the config?

Reply:
its a smaller way of writing the logic:
'''if (len(firebase.apps.length) == 0) {
firebase.initializeApp(firebaseConfig)} 
else {
firebase.app()}'''

in terms of what it does: 
basically, if an app is already loaded then it doesn't init the app, which is the case after the first load.